module/extension/FileWatcher/src/FileWatcher.cpp

```diff
@@ -110,9 +110,9 @@ namespace module::extension {
     FileWatcher::FileWatcher(std::unique_ptr<NUClear::Environment> environment)
         : Reactor(std::move(environment))
         , loop(std::make_unique<uv_loop_t>())
-        , add_watch(std::make_unique<uv_async_t>())
```

Comment:
Not a fan of new...
I think I'd rather either a custom wrapper struct for these, or shared_ptrs so you can set the deleter per object